app/models/claim.rb

```diff
@@ -35,15 +35,15 @@ class Claim < ApplicationRecord
     updated_at: false,
     verified_fields: false,
     verify_response: true,
-    approved_at: false,
-    approved_by: false,
```

Comment:
Should we have some `checks` related fields in here?

Reply:
I think these are only attributes on the `Claim` model, we use this to stop sensitive data from being logged.

Reply:
We have `eligibility_id` and `eligibility_type` in there, too, which makes me think that the thing checking for these isn't working?

Reply:
It looks like the `accepts_nested_attributes_for` is adding `eligibility_id` and `eligibility_type` to the attributes for the Claim, I am not sure we would access the Check attributes in the same way?

Reply:
Ok, makes sense :).